docs/code_examples/systemd.service

```diff
@@ -4,6 +4,7 @@ After=network.service
 
 [Service]
 Environment="LC_ALL=en_US.UTF-8"
+Environment="PATH=/usr/bin:/path/to/errbot/virtualenv/bin"
```

Comment:
We should probably set `/bin/` if we're setting `$PATH` here.

Reply:
maybe add the full default path ? like `/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin`

Reply:
The term "default" is a bit subjective and OS-vendor dependent, but that seems sensible yes.